query_postal_codes_heatmaps_api.py

```diff
@@ -0,0 +1,104 @@
+from oauthlib.oauth2 import BackendApplicationClient
+from requests_oauthlib import OAuth2Session
+from datetime import datetime, timedelta, date
+import matplotlib.pylab as plt
```

Comment:
You probably want `import matplotlib.pyplot as plt` as documented [here](https://matplotlib.org/3.1.1/tutorials/introductory/usage.html#matplotlib-pyplot-and-pylab-how-are-they-related):
> pylab is deprecated and its use is strongly discouraged because of namespace pollution. Use pyplot instead.